src/langchain_google_alloydb_pg/engine.py

```diff
@@ -352,6 +452,7 @@ def init_vectorstore_table(
         )
 
     async def ainit_chat_history_table(self, table_name: str) -> None:
+        """Create a new chat history table."""
```

Comment:
Can you expand this to be similar to `init_vectorstore_table`

Reply:
expanded